ucm2/USB-Audio/Behringer/UMC202HD-HiFi.conf

```diff
@@ -0,0 +1,116 @@
+Include.pcm_split.File "/common/pcm/split.conf"
+
+Macro [
+	{
+		SplitPCM {
+			Name "umc202hd_stereo_out"
+			Direction Playback
+			Channels 2
+			HWChannels 2
+			HWChannelPos0 FL
+			HWChannelPos1 FR
+		}
+	}
+	{
+		SplitPCM {
+			Name "umc202hd_mono_in"
+			Direction Capture
+			Channels 1
+			HWChannels 2
+			HWChannelPos0 MONO
+			HWChannelPos1 MONO
+		}
+	}
+]
+
+Define.PCMCTLName "PCM Playback"
+
+If.pcm_controls {
+	Condition {
+		Type ControlExists
+		Control "name='UMC202HD 192k Output Playback Switch'"
+	}
+	True.Define.PCMCTLName "UMC202HD 192k Output Playback"
+}
+
+Include.ctl_remap.File "/common/ctl/remap.conf"
+
+Macro [
+	{
+		CtlRemapStereoVolSw {
+			Dst "Line A Playback"
+			Src "${var:PCMCTLName}"
+			Index0 0
+			Index1 1
+		}
+	}
+	{
+		CtlRemapMonoVolSw {
+			Dst "Input 1 Capture"
+			Src "Mic Capture"
+			Index 0
+		}
+	}
+	{
+		CtlRemapMonoVolSw {
+			Dst "Input 2 Capture"
+			Src "Mic Capture"
+			Index 1
+		}
+	}
+]
+
+SectionDevice."Line1" {
+	Comment "Line A"
+	Value {
+		PlaybackPriority 200
+		PlaybackMixer "default:${CardId}"
+		PlaybackMixerElem "Line A"
```

Comment:
Change this to `PlaybackMixerElem "UMC202HD 192k Output"` resp. `PlaybackMixerElem "${var:PCMPlaybackElem}"` and define `PCMPlaybackElem` variable.